Header header

uint16 height
uint16 width

uint8 detected_objects
uint8 foveation_level

FoveatedImages[] foveated_images_groups